src/npm.js

```diff
@@ -107,6 +103,33 @@ function runYarn(deps, options, exec) {
 	});
 }
 
+/**
+ * Add version or latest to package name
+ * @param {string[]} deps
+ * @param {string[]} versions
+ */
+function getVersionedDep(versions) {
```

Comment:
JSDoc does't match the actual implementation, and I think the JSDoc version is better ;-)

```js
function getVersionedDep(dep, versions) {
  const version = versions[dep] || 'latest';
  return `${dep}@${version}`;
}

newDeps.map(dep => getVersionedDep(dep, versions))
```

Reply:
Ah, actually I used currying to simplify the usage in map, but up to you , result will be the same

Reply:
Changed 👍